python/ray/_private/utils.py

```diff
@@ -2008,6 +2009,14 @@ def pasre_pg_formatted_resources_to_original(
     original_resources = {}
 
     for key, value in pg_formatted_resources.items():
+        # Filter out resources that have bundle_group_[pg_id] since
+        # it is an implementation detail.
+        # This resource is automatically added to the resource
+        # request for all tasks that require placement groups.
+        result = PLACEMENT_GROUP_BUNDLE_KEY_RESOURCE_PATTERN.match(key)
```

Comment:
Nit: I don't think we need to introduce another regex, We can just skip if the parsed original resource == "bundle"

Reply:
Probably not in this PR but ideally we should do the same thing for Java and C++ as well, otherwise they will see the bundle resource.

Reply:
good point. Fixed it, and I also added input validation to disallow "bundle"

Reply:
> dded input validation to disallow "bundle"

Will this break current users if they don't use PG. Maybe we should rename our "bundle" to something more private like `_pg_bundle_`